docs/src/administration/users.md

```diff
@@ -1,138 +1,162 @@
+To delete a user from a project, follow these steps:
 
-```bash
-platform user:add alice@example.com -r admin
-```
+{{< codetabs >}}
+---
+title="In the console"
+---
+1. Go to your console and select the project where you want to delete a user.
+2. Under **Settings**, click **Access**.
+3. Select the user you want to delete and click **Delete**
+4. Click **Save**.
+5. After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by clicking **Redeploy** in the console.
+---
+title="Using the CLI"
+---
+Use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md) to delete existing users.
+
+* `platform user:delete`
 
-After adding `alice@example.com`, Alice will receive an invitation email asking to confirm her details and optionally, register for a Platform.sh account.
+  ```bash
+  platform user:delete user1@example.com
+  ```
 
-To give Bob different levels of access depending on the environment type, in the current project:
- 
--  **Viewer** role to the **Production environment**
--  **Contributor** role to **all Staging type environments**
--  **Admin** role to **all Development type environments**
+  Use `platform list` to get the full list of commands.
 
-You would run:
+  After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by using the CLI command platform `redeploy`.
 
-```bash
-platform user:role bob@example.com -r production:viewer -r staging:contributor -r development:admin
-```
+{{< /codetabs >}}
 
-Use `platform list` to get the full list of commands.
+Once you delete a user, they won't be able to access the project anymore.
 
-## User access and integrations
 
-If you have setup an [external integration](/integrations/source/_index.md) to GitHub, GitLab, or Bitbucket, this adds an additional layer of access control to the project that you will need to be aware of.
+### Change user permissions from the console
 
-For example, if you invite a user with **Project Admin** role to a project on Platform.sh, but you haven't invited them to the remote repository on GitHub / GitLab, they might be [unable to clone the project](/administration/web/_index.md#git) locally.
+To change the user permissions, follow the steps:
 
-In this example, using either `platform get` with the CLI:
+{{< codetabs >}}
+---
+title="In the console"
+---
+1. Go to your console and select the project where you want to change the user permissions.
+2. Under **Settings**, click **Access**.
+3. Select a user and change the permissions.
+4. Click **Save**.
+5. After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by clicking **Redeploy** in the console.
```

Comment:
As the console notes, this isn't actually a necessary step:
![image](https://user-images.githubusercontent.com/26428882/136815656-da7b8525-b778-4fa5-81a4-e45b2b1a4a3f.png)
It seems it's done automatically.

It should probably be remove, but at least changed to not being about deleting users.